modules/publication/jsx/viewProject.js

```diff
@@ -25,16 +25,6 @@ class ViewProject extends React.Component {
 
   validateEmail(field, email) {
     let formErrors = this.state.formErrors;
-
-    // don't supply error if email is blank
```

Comment:
what has this validation been replaced with?

Reply:
hey @maltheism, what's your solution for this? the validation is still missing

Reply:
Hey @zaliqarosli I'm not sure why the validateEmail is showing here on GitHub. I've removed the validateEmail function here if you checkout the PR and look at the code. The validation for the email is being done in the uploadForm.js

Reply:
I know that you've removed the validateEmail function here. my question is why? shouldn't we keep the validation here?

it looks to me that validateEmail prop is to be supplied to the `<ProjectFormFields />` component of which there are two - one in `viewProject.js` and one in `uploadForm.js`. it looks to me that validateEmail function needs to be defined in both` viewProject.js` and `uploadForm.js`.

You are removing the validateEmail function from `viewProject.js`. If it looks to you that `viewProject.js` doesn't need the validateEmail function, please explain to me why. And explain to me why the `<ProjectFormFields />` component in this same class doesn't need the validateEmail prop (which you are also removing)

Reply:
@zaliqarosli Thanks for all the help!